Connectors/Rhino/Speckle.Connectors.RhinoShared/Operations/Receive/RhinoHostObjectBuilder.cs

```diff
@@ -187,7 +187,7 @@ CancellationToken cancellationToken
             if (conversionIds.Count == 0)
             {
               // TODO: add this condition to report object - same as in autocad
-              throw new SpeckleException($"Failed to convert object.");
```

Comment:
did you check messaging of other connectors? curious why you only interested with rhino for ifc

Reply:
I didn't rigorously test this in all connectors, but only rhino appears to have such a check.